src/main/java/org/example/Services/CustomerServices.java

```
@@ -12,41 +14,50 @@
 
     public CustomerServices() throws SQLException {
     }
 
-    public void depositMoney(String customerName, int depositAmount) throws IOException, SQLException {
+    public void depositMoney(String customerName, int depositAmount) throws SQLException {
 
         int returnedId = dbcontroller.findCustomerByUsername(customerName);
         Customer customer = dbcontroller.returnRecord(returnedId);
         int oldAmount = customer.moneyInCents();
         int newAmount = transactions.receiveMoneyTransaction(oldAmount, depositAmount);
         boolean fundsupdated = dbcontroller.updateCustomerFunds(returnedId, newAmount);
         if (fundsupdated){
-            System.out.println(customerName + " funds were updated!");
+            System.out.println(customerName + " money was updated!");
+            logger.info("{} money was updated by {}! ", customerName, depositAmount);
         }
     }
 
-    public boolean makeTransfer(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws SQLException, IOException {
+    public void makeTransfer(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws SQLException, IOException {
         int senderreturnedId = dbcontroller.findCustomerByUsername(nameSenderCustomer);
-        int receiverreturnedId = dbcontroller.findCustomerByUsername(nameReceiverCustomer);
+        int receiverreturnedId = dbcontroller1.findCustomerByUsername(nameReceiverCustomer);
 
-        boolean sendmoney = sendMoneyToSomeone(senderreturnedId, amountToSend);
-        if (sendmoney){
-            receiveMoney(receiverreturnedId, amountToSend);
-            updateDbAfterTransfer(senderreturnedId, nameSenderCustomer, receiverreturnedId, nameReceiverCustomer, amountToSend);
-            return true;
-        }
-        return false;
-    }
+        boolean moneyReceived = false;
+
+            boolean sendmoney = sendMoneyToSomeone(senderreturnedId, amountToSend);

```

Shouldn't use true/false returns like this. Another developer will not know the meaning behind true/false returns unless it's some sort of check (eg.: `isMoneySent(...)`). Throw an error if this operation is unsuccessful instead of returning false.

---

it exactly is 'isMoneySent' check, just bad naming habits